test/api_system_metrics_test.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"regexp"
+
+	"github.com/alibaba/pouch/pkg/kernel"
+	"github.com/alibaba/pouch/test/environment"
+	"github.com/alibaba/pouch/version"
+
+	"github.com/go-check/check"
+)
+
+// APIEngineVersionMetricsSuite is the test suite for EngineVersion metrics API.
+type APIEngineVersionMetricsSuite struct{}
+
+func init() {
+	check.Suite(&APIEngineVersionMetricsSuite{})
+}
+
+// SetUpTest does common setup in the beginning of each test.
+func (suite *APIEngineVersionMetricsSuite) SetUpTest(c *check.C) {
+	SkipIfFalse(c, environment.IsLinux)
```

Comment:
please remove the setuptest and teardownsuite because the setupsuite is good enough

Reply:
ok, already deleted